cmake/modules/llext.cmake

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2024 Schneider Electric
+# SPDX-License-Identifier: Apache-2.0
+
+include_guard(GLOBAL)
+
+message("In llext cmakelists.txt")
```

Comment:
?

Reply:
just logging what's happening when you actually build an extension. I can remove it.

Reply:
@nordicjm Please elaborate a bit on what you mean by "?", its far from helpful or explanatory of what you are looking for and why.

Reply:
yep, got it and remove it